components/infobox/wikis/apexlegends/infobox_character_custom.lua

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Blank lines where applicable

Reply:
I'm not sure I understand?

Reply:
Like https://github.com/Liquipedia/Lua-Modules/pull/1707/commits/7479f29abab70df5cb1be26d7bc61e44d63ccaea